src/applications/personalization/profile-2/tests/reducers.unit.spec.js

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai';
+
+import reducer from '../reducers';
+import * as actions from '../actions';
+
+describe('Profile reducer', () => {
+  let state;
+  let reducedState;
+  let action;
+
+  beforeEach(() => {
+    state = undefined;
+  });
+
+  describe('OPEN_SIDE_NAV', () => {
+    it('sets `isSideNavOpen` to `true` and `focusTriggerButton` to `false`', () => {
+      action = {
+        type: actions.OPEN_SIDE_NAV,
+      };
+      reducedState = reducer(state, action);
+      expect(reducedState.isSideNavOpen).to.be.true;
+      expect(reducedState.focusTriggerButton).to.be.false;
+    });
+  });
+
+  describe('CLOSE_SIDE_NAV', () => {
+    it("sets `isSideNavOpen` to `false` and sets `focusTriggerButton` to the action's value", () => {
```

Comment:
What string values does `focusTriggerButton` take?

Reply:
Well it _should_ took a boolean. But this is just testing that the value of `action.focusTriggerButton` gets shoved into the state for the value of `focusTriggerButton`.